_script/setup

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2021 Adevinta
```

Comment:
s/2021/2023

Reply:
Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034